holoviews/plotting/bokeh/plot.py

```diff
@@ -40,33 +38,8 @@ class BokehPlot(DimensionedPlot):
     plotting interface for Bokeh based plots.
     """
 
-    width = param.Integer(default=300, doc="""
-        Width of the plot in pixels""")
-
-    height = param.Integer(default=300, doc="""
-        Height of the plot in pixels""")
-
-    sizing_mode = param.ObjectSelector(default='fixed',
-        objects=['fixed', 'stretch_both', 'scale_width', 'scale_height',
-                 'scale_both'], doc="""
-        How the item being displayed should size itself.
-
-        "stretch_both" plots will resize to occupy all available
-        space, even if this changes the aspect ratio of the element.
-
-        "fixed" plots are not responsive and will retain their
-        original width and height regardless of any subsequent browser
-        window resize events.
-
-        "scale_width" elements will responsively resize to fit to the
-        width available, while maintaining the original aspect ratio.
-
-        "scale_height" elements will responsively resize to fit to the
-        height available, while maintaining the original aspect ratio.
-
-        "scale_both" elements will responsively resize to for both the
-        width and height available, while maintaining the original
-        aspect ratio.""")
+    css_classes = param.List(default=[], doc="""
+        CSS classes to apply to the layout.""")
```

Comment:
As far as I can tell there are no documented examples of how to use this plot option... (but maybe there is a diff I haven't expanded in this PR somewhere)

Reply:
I'll add a small example to the bokeh guide, it's mostly useful for server apps I would think but adding CSS in a notebook is straightforward enough.

Reply:
Tbh, I'm probably going to remove it, I haven't been able to come up with a useful example. It makes sense for Row and Column containers once we add them but at the plot level messing with css just doesn't seem useful.